threedi_models_and_simulations/widgets/build_options.py

```diff
@@ -86,3 +87,27 @@ def download_schematisation(self):
             if wip_revision is not None:
                 settings = QSettings("3di", "qgisplugin")
                 settings.setValue("last_used_spatialite_path", wip_revision.schematisation_dir)
+
+    @api_client_required
+    def load_schematisation(self, schematisation, revision):
```

Comment:
Name of this method is quite generic and might be missleading. Could you please rename it to something like `load_remote_schematisation` ?